resources/model/OpenStudio.idd

```diff
@@ -29887,6 +29887,59 @@ OS:ZoneHVAC:IdealLoadsAirSystem,
        \minimum 0.0
        \maximum 1.0
        \default 0.65
+  A16, \field Design Specification ZoneHVAC Sizing Object Name
+       \note Enter the name of a DesignSpecificationZoneHVACSizing object.
+       \type object-list
+       \object-list DesignSpecificationZoneHVACSizingName
```

Comment:
So this is just a reservation because we don't have this object right?

why don't we have it? Isn't it weird that no one asked for it? It was added in E+ V8-2-0